bridge_adaptivity/module/views.py

```diff
@@ -406,3 +406,17 @@ def callback_sequence_item_grade(request):
         policy.send_grade()
 
     return HttpResponse(xml, content_type="application/xml")
+
+
+def sync_collection(request, pk):
+    """
+    Synchronize collection immediately.
+    """
+    back_url = request.GET.get('back_url')
+    collection = Collection.objects.get(pk=pk)
```

Comment:
probably it should be better to use  `get_object_or_404` here.

Reply:
agree, nice catch!

Reply:
done